Fastlane/Fastfile

```diff
@@ -87,92 +98,74 @@ lane :test_project do |options|
 end
 
 desc 'Tests the WeTransferPRLinter Swift Package'
-lane :test_pr_linter do |options|
+lane :test_pr_linter do
 	spm(command: "generate-xcodeproj", package_path: "Danger-Swift")
 	test_project(
-		project_path: "Danger-Swift/", 
-		project_name: "WeTransferPRLinter", 
+		project_path: "Danger-Swift/",
+		project_name: "WeTransferPRLinter",
 		scheme: "WeTransferPRLinter-Package",
 		device: nil,
 		destination: "platform=macOS")
 end
 
 desc 'Create a release from a tag triggered CI run'
-lane :release_from_tag do |options|
+lane :release_from_tag do
 	# Get the latest tag, which is the new release that triggered this lane.
 	sh('git fetch --tags origin master --no-recurse-submodules -q')
 
 	latest_tag = ENV["BITRISE_GIT_TAG"]
-	
+
 	# Create a release branch
 	sh "git branch release/#{latest_tag} origin/master"
-    sh "git checkout release/#{latest_tag}"
-    sh "git merge -X theirs #{latest_tag}"
+  sh "git checkout release/#{latest_tag}"
+  sh "git merge -X theirs #{latest_tag}"
 
 	release_output = sh('gitbuddy release -c "../Changelog.md"')
 	release_url = URI.extract(release_output).find { |url| url.include? 'releases/tag' }
-    puts "Created release with URL: #{release_url}"
+  puts "Created release with URL: #{release_url}"
 
 	# Run only if there's a podspec to update
 	if Dir['../*.podspec'].any?
 		# Update the podspec. It finds the .podspec automatically in the current folder.
 		version_bump_podspec(version_number: latest_tag)
-		
+
 		begin
 			# Push the podspec to trunk
 			pod_push
 		rescue => ex
 			UI.important("Pod push failed: #{ex}")
 		end
-	end 
+	end
 
 	# Push the changes to the branch
 	sh('git commit -a -m "Created a new release"')
-    sh("git push origin release/#{latest_tag}")
-	
-	# Create a PR
-    # Create a pull request for master to include the updated Changelog.md and podspec
-    create_pull_request(
-      api_token: ENV["DANGER_GITHUB_API_TOKEN"],
-      repo: git_repository_name,
-      title: "Merge release #{latest_tag} into master",
-      base: "master", # The branch to merge the changes into.
-      body: "Containing all the changes for our [**#{latest_tag} Release**](#{release_url})."
-    )
+  sh("git push origin release/#{latest_tag}")
```

Comment:
It looks like indenting is a bit off on some places

Reply:
Yes it is not aligned anymore. It happened in more places, therefore I used rubicon to lint all the files.